config/config-server/resources/cruise-config.xsd

```diff
@@ -393,6 +393,7 @@
         <xsd:attribute name="url" type="xsd:string" use="required"/>
         <xsd:attribute name="dest" type="filePathType" use="optional"/>
         <xsd:attribute name="branch" type="xsd:string" use="optional"/>
+        <xsd:attribute name="shallowClone" type="xsd:boolean" use="optional"/>
```

Comment:
This is a pure "addition" to the scheme. Do I need bump up config schema version?